pyproject.toml

```
@@ -0,0 +1,26 @@
+[tool.black]
+line-length = 88
+target_version = ['py36']
+exclude = '''
+/(
+    venv
+  | \.git
+  | \.hg
+  | \.mypy_cache
+  | \.tox
+  | \.venv
+  | _build
+  | buck-out
+  | build
+  | dist
+
+  # The following are specific to Black, you probably don't want those.
+  | blib2to3
+  | tests/data
+)/
+'''
+
+[tool.isort]
+line_length=88
+multi_line_output=3
+include_trailing_comma=true
```

Using an [`isort` profile](https://pycqa.github.io/isort/docs/configuration/profiles/) simplifies the configuration & eliminates some weird edge cases where isort and black format differently.

```suggestion
profile = "black"
```